keyrings/alt/file.py

```diff
@@ -78,9 +81,9 @@ def priority(self):
             __import__('Crypto.Cipher.AES')
             __import__('Crypto.Protocol.KDF')
             __import__('Crypto.Random')
-        except ImportError:
+        except ImportError:     # pragma: no cover
```

Comment:
Hmm, any idea of how to test these exceptions, e.g. how to raise these ImportError exceptions, while Crypto _is_ installed? Is that mockable?